### ISSUE TITLE
Need for an .editorconfig and CONTRIBUTING.md

### ISSUE THREAD

**Issue by thblckjkr** [emmabostian/developer-portfolios#125]
As @eclectic-coding pointed on [this](https://github.com/emmawedekind/developer-portfolios/pull/75#issuecomment-567227017) comment, we should have a contribution guidelines for the portfolios, and also we should consider to have a .editorconfig to prevent more issues with spacing.

Any idea or suggestion on how to implement it?

**Comment by thblckjkr** [comment]
BTW, a classification by type, or something could be useful for the portfolios.

Just general classifications for easiness finding portfolios about different topics

**Comment by geekskick** [comment]
The contributing.md should also have a command line expression or something which will ensure the names are in alphabetical order. Maybe there is a need for a custom test.py that formats according to the style and also checks for dead/incorrect links.

**Comment by eclectic-coding** [comment]
Great conversation. The first repo I created a PR for used a variation of the following [Contribution](https://github.com/eclectic-coding/Modulus-Pro/blob/develop/CONTRIBUTING.md) file. This type of file made it easier for me as a new user. This specific file is on a project I did a few years ago and may be a little more than we are looking for, but it may provide a good starting point. Thoughts?
